po/ca.po

```diff
@@ -1342,17 +1307,18 @@ msgid "%s: Cannot setup cleanup service.\n"
 msgstr "%s: no es pot configurar el servei de neteja.\n"
 
 msgid ""
-"  -P, --prefix PREFIX_DIR       prefix directory where are located the /etc/"
-"* files\n"
+"  -P, --prefix PREFIX_DIR       prefix directory where are located the /etc/* "
+"files\n"
 msgstr ""
+"  -P, --prefix PREFIXDIR directori de prefixos on es troben els fitxers /etc/"
+"*\n"
```

Comment:
Jo dividiria estes llinies aixina:

"  -P, --prefix PREFIX_DIR       "
"prefix directory where are located the /etc/* files\n"

(Per a tots els casos similars tambe: dividir just on comenca la descripcio.)

Reply:
Vols dir de modificar el "msgid" perquè quedi tal com ensenyes?
I el "msgstr" també evidentment...

Ho començaré a fer en un commit nou, en cas que no ens entenem (trobo estrany modificar el "msgid" extret, ja veig que té sentit però amb la lògica de separar línies semànticament)

Reply:
He separat les línies msgid i msgstr dels paràmetres de cadenes que he traduït. Puc fer squash si és el que volem.

Fa molt de temps que no traduia res, però poedit sembla força utilitzat:

```
carles@pinux:[update-catalan-translation]~/git/shadow-upstream/po$ grep -i "x-generator: poedit" *
grep: ca.mo: binary file matches
ca.po:"X-Generator: Poedit 3.2.2\n"
fr.po:"X-Generator: Poedit 2.4.2\n"
ka.po:"X-Generator: Poedit 3.3.2\n"
kk.po:"X-Generator: Poedit 1.8.11\n"
nb.po:"X-Generator: Poedit 1.7.5\n"
nl.po:"X-Generator: Poedit 2.2.1\n"
zh_CN.po:"X-Generator: Poedit 2.0.8\n"
carles@pinux:[update-catalan-translation]~/git/shadow-upstream/po$ 
```
I poedit reformateja la separació de les cadenes.

Ja sé que és responsabilitat de cada un d'utilitzar eines que s'adeqüin, però tinc dubtes si a llarg termini fer formateig manual de les cadenes pot ser contra-produent.

Veig que re-formateja tant "msgid" com "msgstr" (però mai havia vist que el "msgid" extret i reformatejat per poedit fos diferents... però ja et dic, fa temps i temps que no feia traduccions i tampoc és que n'hagués fet un munt)

Parlo de poedit perquè és el que faig servir. Altres .po son amb kbabel, que sembla que el substitut és lokalize. lokalize fa el mateix i reformateja línies.

Ho dic perquè tinc dubtes que reformatejar línies sigui bona idea, si després el proper traductor no pot fer servir eines comunes per assistir a la feina.

Reply:
Jo tenia en ment soles els msgstr.  Els msgid, si son autogenerats millor no tocar-los.

Reply:
Encara que com tu veges.  Si creus que es millor com estava, aixina ho deixem.

Reply:
Jo crec que deixar-ho pels motius que he dit. És més, penso que en regenerar els .po quan hi ha noves cadenes el formateig del msgstr es trencaria.

Havia detectat tres errors:
- _canvia -> canvia (que just has comentat)
- "prefixos de directoris" (és incorrecte)
- extra new line: no es pot utilitzar amb --prefix\n" 

Ho arreglaré. Miraré abans que al regenerar els .po els msgstr es reformateixen com em temo perquè tinc curiositat...

Reply:
Efectivament, al fer "make update-po" tot el formateig tant de msgid com de msgstr es perd. Part de git diff després de make update-po:

```
 #, c-format
-msgid "%s: Can't get unique system GID (%s). Suppressing additional messages.\n"
+msgid ""
+"%s: Can't get unique system GID (%s). Suppressing additional messages.\n"
 msgstr ""
-"%s: no es pot obtenir un GID de sistema únic (%s). "
-"S'estan suprimint els missatges addicionals.\n"
+"%s: no es pot obtenir un GID de sistema únic (%s). S'estan suprimint els "
+"missatges addicionals.\n"
```

Així que desfaré aquest canvi que es perd en fer actualitzacions de cadenes (o si un traductor fa servir poedit/lokalize, etc..) i arreglaré tres strings.